<launch>

    <!-- Node to publish images from airsim client node -->
    <!-- <include file="$(find airsim_ros_client)/launch/pubImages.launch"/> -->

    <!-- Nodelet manager for this pipeline -->
    <node pkg="nodelet" type="nodelet" args="manager" name="depth_to_pointcloud_manager" output="screen"/>

     <!-- Convert to point cloud -->

    	<!-- <node pkg="nodelet" type="nodelet" name="airsim_depth2cloud_colored" args="load depth_image_proc/point_cloud_xyzrgb depth_to_pointcloud_manager" output="screen">
		<remap from="depth_registered/image_rect" to="/camera/depth_registered/image_raw"/>
		<remap from="depth_registered/points" to="/airsim_node/drone_1/points/colored"/>
		<remap from="rgb/image_rect_color" to="/camera/rgb/image_rect_color"/>
		<remap from="rgb/camera_info" to="/camera/rgb/camera_info"/>
	</node> -->

        <node pkg="nodelet" type="nodelet" name="airsim_depth2cloud" args="load depth_image_proc/point_cloud_xyz depth_to_pointcloud_manager" output="screen">
		<remap from="image_rect" to="/camera/depth_registered/image_raw"/>
		<remap from="points" to="/airsim_node/drone_1/points"/>
		<remap from="camera_info" to="/camera/depth_registered/camera_info"/>
	</node>

</launch>